### ISSUE TITLE
Notification: Expose notification mechanism to users

### ISSUE THREAD

**Issue by jrobinAV** [Avaiga/taipy#405]
- [ ] Expose register and unregister methods and add Reference manual documentation

The documentation is covered by ticket [doc-#416](https://github.com/Avaiga/taipy-doc/issues/416).

**Comment by jrobinAV** [comment]
Blocked by [doc-#709](https://github.com/Avaiga/taipy-doc/issues/709).

**Comment by toan-quach** [comment]
@jrobinAV I believe we already have everything we need for this ticket and only need to do some documentation regarding the user manual. I think the reference document is already done as well.

**Comment by jrobinAV** [comment]
OK, that's great.

Maybe we can add a usage example on the Ref man of the Notifier class or maybe on the register/unregister methods.
